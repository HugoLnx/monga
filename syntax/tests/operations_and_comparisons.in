int main() {
	x = k < a * -b > c * d;
}
