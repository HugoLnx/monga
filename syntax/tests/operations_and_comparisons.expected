TK_MINUS exp
exp TK_ASTERISK exp
exp TK_LESS exp
exp TK_ASTERISK exp
exp TK_GREATER exp
var TK_ONE_EQUAL exp
